src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -91,11 +93,50 @@ interface PostLogConfig = {
 ```
 
 #### simulcastEnabled
+
 To enable simulcast for the meeting pass the `simulcastEnabled` prop with value set to `true`. By default, it is `false`.
 For more information on Simulcast, check Amazon Chime JS SDK [simulcast guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/05_Simulcast.md).
 
+#### logger
+
+The `Logger` object that you want to be used in the meeting session.
+If you pass in a `Logger` object using this parameter, the `MeetingManager` will use this object instead of creating a logger based on `logLevel` and `postLogConfig` to initialize the meeting session.
+
+#### videoDownlinkBandwidthPolicy
+
+The `VideoDownlinkBandwidthPolicy` object that you want to be used in the meeting session.
+For more information on `VideoDownlinkBandwidthPolicy`, check Amazon Chime JS SDK [priority downlink policy guide](https://aws.github.io/amazon-chime-sdk-js/modules/prioritybased_downlink_policy.html).
+
+For example, to use the `VideoPriorityBasedPolicy` in the meeting:
+
+```jsx
+import {
+  LogLevel,
+  ConsoleLogger,
+  VideoPriorityBasedPolicy,
+} from 'amazon-chime-sdk-js';
+import { MeetingProvider } from 'amazon-chime-sdk-component-library-react';
+
+const Root = () => {
+  const logger = new ConsoleLogger('SDK', LogLevel.info);
```

Comment:
Just to confirm but have you tried this sample code out and make sure it is working?

Reply:
Yes, I packed the local sdk and installed it in the meeting demo in aws sapmle. It works. There is a typo here, the `LogLevel.info` should be `LogLevel.INFO`, addressed.